### PR TITLE
285 metadata tutorial

### PR DESCRIPTION
**Pull request by chStaiger**
Adjusting the metadata tutorial to the new `MetaData` and `MetaDataItem` classes.

### CODE REVIEW

File: tutorials/04-Metadata.ipynb [hunk 1]
```diff
@@ -237,88 +294,145 @@
...
    "cell_type": "code",
    "execution_count": null,
-   "id": "97b93549",
+   "id": "0ea21f51-42ad-40e4-af92-2227415d5f72",
    "metadata": {},
    "outputs": [],
    "source": [
-    "obj.meta.delete('key', 'value')\n",
-    "print(obj.meta)"
+    "!obj_meta.add('Author', 'Christine')\n",
```
Comment (qubixes) [inline tutorials/04-Metadata.ipynb:375]: This doesn't look right?

File: tutorials/04-Metadata.ipynb [hunk 2]
```diff
@@ -237,88 +294,145 @@
...
   },
   {
    "cell_type": "markdown",
-   "id": "c3f54857",
+   "id": "ad30990d-5e77-4d52-b22b-80d57585351e",
+   "metadata": {},
+   "source": [
+    "### Deleting a single metadata item\n",
+    "\n",
+    "To delete a single metadata item you will have to be again specific with your key, value and units information to identify the correct metadata item. To delete the metadata with the key `Key` we can simply use:"
```
Comment (qubixes) [inline tutorials/04-Metadata.ipynb:386]:
```suggestion
    "To delete a single metadata item you will have to be again specific with your key, value and units information to identify the correct metadata item. To delete all metadata with the key `Key` we can simply use:"
```

**Review by qubixes — CHANGES_REQUESTED** [review]
Thanks for the PR! Some small issues I have found. Just a small request not to use my real name.

File: tutorials/04-Metadata.ipynb (hunk 1 above)
Comment (chStaiger) [inline tutorials/04-Metadata.ipynb]:
```suggestion
    "obj_meta.add('Author', 'Christine')\n",
```

**Pull request merged**